inference-engine/tests/functional/plugin/cpu/single_layer_tests/convolution_backprop_data.cpp

```diff
@@ -212,39 +401,51 @@ const auto convParams_ExplicitPadding_Planar_3D = ::testing::Combine(
 
 INSTANTIATE_TEST_SUITE_P(smoke_Deconv_3D_Planar_FP32, DeconvolutionLayerCPUTest,
     ::testing::Combine(
-        ::testing::Combine(
-            convParams_ExplicitPadding_Planar_3D,
-            ::testing::Values(Precision::FP32),
-            ::testing::Values(Precision::UNSPECIFIED),
-            ::testing::Values(Precision::UNSPECIFIED),
-            ::testing::Values(Layout::ANY),
-            ::testing::Values(Layout::ANY),
-            ::testing::Values(std::vector<size_t >({ 2, 12, 7, 7, 7 })),
-            ::testing::ValuesIn(emptyOutputShape),
-            ::testing::Values(CommonTestUtils::DEVICE_CPU)),
-        ::testing::ValuesIn(filterCPUInfoForDevice({conv_gemm_3D})),
+        convParams_ExplicitPadding_Planar_3D,
+        ::testing::ValuesIn(Planar_3D_inputs),
+        ::testing::Values(ElementType::f32),
         ::testing::ValuesIn(fusingParamsSet),
         ::testing::Values(cpuEmptyPluginConfig)),
     DeconvolutionLayerCPUTest::getTestCaseName);
 
 INSTANTIATE_TEST_SUITE_P(smoke_Deconv_3D_Planar_BF16, DeconvolutionLayerCPUTest,
     ::testing::Combine(
-        ::testing::Combine(
-            convParams_ExplicitPadding_Planar_3D,
-            ::testing::Values(Precision::FP32),
-            ::testing::Values(Precision::BF16),
-            ::testing::Values(Precision::BF16),
-            ::testing::Values(Layout::ANY),
-            ::testing::Values(Layout::ANY),
-            ::testing::Values(std::vector<size_t >({ 2, 12, 7, 7, 7 })),
-            ::testing::ValuesIn(emptyOutputShape),
-            ::testing::Values(CommonTestUtils::DEVICE_CPU)),
-        ::testing::ValuesIn(filterCPUInfoForDevice({conv_gemm_3D})),
+        convParams_ExplicitPadding_Planar_3D,
+        ::testing::ValuesIn(Planar_3D_inputs),
+        ::testing::Values(ElementType::f32),
         ::testing::ValuesIn(fusingParamsSet),
         ::testing::Values(cpuBF16PluginConfig)),
     DeconvolutionLayerCPUTest::getTestCaseName);
 
-/* ============= GroupDeconvolution (Blocked 2D) ============= */
+const auto deconv_blk_ref_2D = CPUSpecificParams{{nChw16c}, {nChw16c}, {"ref_any"}, "ref_any"};
```

Comment:
> Considering the last changes aimed to enable fallback on performance implementation, such combinations should be executed with gemm implementation, shouldn't they?
At this moment fallback on reference occur only if we can't create implementation which equal selected implementation type.
But anyway if it occurred we still got primitive type as `ref_any` because when we dump execution graph we take implementation type from `selectedPrimitiveDescriptor`. https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/src/mkldnn_plugin/mkldnn_node.cpp#L375

> Moreover, previous static tests used avx512 implementation, and now it seems it is not tested anymore. Am I rights?
For static case we still executed `avx512` implementation, please see the first element in `Blocked_*D_inputs`

Reply:
Got it. So does it mean that in the dynamic case we can not count for an optimized implementation surrounded with reorders and we always get `ref_any` since it was selected on the graph initialization stage and in roughly all cases this implementation is available. Do we have any insights on the performance of `ref_any` vs optimized+reorders?

Reply:
measured on test: `IS={?.35.?.?.?}_TS=((1.35.5.5.5))_((2.35.5.7.5))_PRC=f32_K=(3.3.3)_S=(2.2.2)_PB=(0.0.0)_PE=(0.0.0)_D=(1.1.1)_OP=()_O=32_AP=explicit_OUT_SH=PARAMETER_OUT_D=((7.7.7))_((7.9.7))__inFmts=nCdhw16c_outFmts=nCdhw16c`
reorder + jit_gemm + reorder: 1st run: 16867 ms 2nd run: 224 ms
ref_any: 1st run: 7004 ms 2nd run: 18144 ms